pkg/model/model.go

```diff
@@ -296,8 +344,61 @@ func (m *Model) GetCRDs() ([]*CRD, error) {
 		// Now add the additional printer columns that have been defined explicitly
 		// in additional_columns
 		crd.addAdditionalPrinterColumns(m.cfg.GetAdditionalColumns(crdName))
-
 		crds = append(crds, crd)
+
+		for _, toInjectNestedFields := range alltoInjectNestedFields {
+
+			addThis := ""
+			var previous *Field
+			var res bool
+
+			len := len(toInjectNestedFields.arr)
+
+			for n, value := range toInjectNestedFields.arr {
+				if n == 0 {
+					// first element is a field of CRD
+					fieldsInCRD := crd.SpecFields
```

Comment:
This could have been declared outside, it's a non necessary operation to do at every loop (same for few variables below)

Reply:
There is a relation between `fieldsInParent` and `previous` variable, which changes with every loop. I can declare the `fieldsInCRD` outside the loop, but that is the only variable that can be defined outside the loop. To maintain the code readability I declared it the same way as `fieldsInParent`.